packages/runtime/test/transport/relationships.test.ts

```diff
@@ -74,6 +74,28 @@ describe("Create Relationship", () => {
         expect(relationship2Response).toBeSuccessful();
         expect(relationship2Response.value.status).toBe("Active");
     });
+
+    test("returns error if templator has active IdentityDeletionProcess", async () => {
+        // create new services that don't have a relationship yet
+        const runtimeServices = await serviceProvider.launch(2, { enableRequestModule: true, enableDeciderModule: true, enableNotificationModule: true });
+        services1 = runtimeServices[0];
+        services2 = runtimeServices[1];
```

Comment:
you never close the runtimeServices. Please wrap this test in a describe block and put the init in a beforeAll and the closing in an afterAll block